### PR TITLE
add api in graph and show on dashboard

### PR DESCRIPTION
**Pull request by amansamyotech**

### CODE REVIEW

File: src/views/Booking/AddItems/ItemAdd.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  const formik = useFormik({
+    initialValues,
+    validationSchema,
+    onSubmit: async (values, { resetForm }) => {
+      const com_url = `${url.base_url}${url.item.create}`;
+
+      const response = await createItem(com_url, values);
+
+      if (response) {
+        toast.success(`Successfully Created`);
```
Comment (aloksamyotech) [inline src/views/Booking/AddItems/ItemAdd.js:30]: put in the constant

File: src/views/Booking/BookingDetails.js [hunk 2]
```diff
@@ -98,135 +94,166 @@ const BookingDetails = ({ open, handleClose, bookingData, onSuccess }) => {
...
+                    fontSize: '0.9rem',
+                    '&:hover': {
+                      borderColor: '#3f51b5'
+                    },
+                    '& .MuiSelect-icon': {
+                      color: '#3f51b5'
+                    }
+                  }}
+                >
+                  {employees.map((employee) => (
```
Comment (aloksamyotech) [inline src/views/Booking/BookingDetails.js:233]: use optional chaining

File: src/views/dashboard/Default/TotalGrowthBarChart.js [hunk 3]
```diff
@@ -39,8 +42,27 @@ const TotalGrowthBarChart = ({ isLoading }) => {
...
   const secondaryLight = theme.palette.secondary.light;
 
+  const fetchMonthViseData = async () => {
+    const com_url = `${url.base_url}${url.dashboard.monthViseBooking}`;
+    const bookingCount = await allBookingCount(com_url);
+
+    var countArray = bookingCount.data.map((item) => item.count);
+    setCountValue(countArray);
+  };
+  console.log(`countValue`, countValue);
```
Comment (aloksamyotech) [inline src/views/dashboard/Default/TotalGrowthBarChart.js:52]: remove console

File: src/views/Booking/AddItems/ItemAdd.js (hunk 1 above)
Comment (amansamyotech) [reply src/views/Booking/AddItems/ItemAdd.js:30]: resolve comment

File: src/views/Booking/BookingDetails.js (hunk 2 above)
Comment (amansamyotech) [reply src/views/Booking/BookingDetails.js:233]: resolve comment

File: src/views/dashboard/Default/TotalGrowthBarChart.js (hunk 3 above)
Comment (amansamyotech) [reply src/views/dashboard/Default/TotalGrowthBarChart.js:52]: resolve comment

**Review by aloksamyotech — APPROVED** [review]

**Pull request merged**